src/lib.rs

```diff
@@ -77,6 +77,9 @@ pub trait Field:
     /// Returns an element chosen uniformly at random using a user-provided RNG.
     fn random<R: RngCore + ?Sized>(rng: &mut R) -> Self;
 
+    /// Returns an element chosen uniformly at random using a user-provided RNG.
+    fn try_from_rng<R: TryRngCore + ?Sized>(rng: &mut R) -> Result<Self, R::Error>;
```

Comment:
Concept ACK (this will address #109), but I want to understand the `?Sized` change first (https://github.com/zkcrypto/ff/pull/126#discussion_r1986209769).

Also, this change to the trait should be documented in the changelog.